src/components/WebexActivityStream/WebexActivityStream.js

```diff
@@ -179,8 +180,16 @@ TimeRuler.propTypes = {
 };
 
 export default function WebexActivityStream({roomID}) {
+  const [activitiesData, dispatch] = useActivityStream(roomID);
+  const loadPreviousActivities = (previousActivities) => {
```

Comment:
why is this here instead of being inside the hook?

Reply:
Because two different hooks need the same function